client/components/Editor/utils/view.ts

```diff
@@ -20,6 +20,14 @@ export const getJSON = (editorView) => {
 export const getTextFromDoc = (doc: Node, separator = '\n') => {
 	return doc.textBetween(0, doc.nodeSize - 2, separator);
 };
+
+export const getImages = (editorView) => {
```

Comment:
I would name this something like `getTopLevelImages`, since images may appear more deeply nested inside the Prosemirror document, and this will not pick those up.